core/storage/blog/gae_models.py

```diff
@@ -76,7 +76,7 @@ class BlogPostModel(base_models.BaseModel):
     # post is a draft.
     thumbnail_filename = datastore_services.StringProperty(indexed=True)
     # Time when the blog post model was last published. Value will be None
-    # if the blog has never been published.
+    # if the blog is not published.
```

Comment:
Perhaps "is not currently published" (to differentiate from "has ever been published"), ditto below in the summary model.

Reply:
Done.